cmd/tendermint/commands/reset_priv_validator.go

```diff
@@ -73,7 +73,10 @@ func ResetAll(dbDir, addrBookFile, privValKeyFile, privValStateFile string, logg
 
 func resetFilePV(privValKeyFile, privValStateFile string, logger log.Logger) {
 	if _, err := os.Stat(privValKeyFile); err == nil {
-		pv := privval.LoadFilePVEmptyState(privValKeyFile, privValStateFile)
+		pv, err := privval.LoadFilePVEmptyState(privValKeyFile, privValStateFile)
+		if err != nil {
+			panic(err)
```

Comment:
Since this is a command, would be nice to bubble up an error. Commands shouldn't really panic.

Reply:
Quickly removed other panics as well.